package.json

```diff
@@ -1,42 +1,42 @@
 {
-  "name": "phaser3-rollup-typescript",
-  "version": "1.1.0",
-  "description": "Phaser 3 TypeScript Starter",
+  "name": "marblaze",
+  "version": "0.0.0",
+  "description": "🎱🔥 wip",
   "main": "dist/index.js",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
     "serve": "vite preview",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
-    "format": "prettier --check 'src/**/*.{js,jsx,ts,tsx,json,css,scss,md}'",
-    "format:fix": "prettier --write 'src/**/*.{js,jsx,ts,tsx,json,css,scss,md}'"
-
+    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
```

Comment:
ok but how about `"lint:fix": "npm run lint -- --fix"` instead

Reply:
i like it, pushed